packages/account-v2/src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ComponentProps, HTMLAttributes, PropsWithChildren, ReactElement } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { Text } from '@deriv-com/ui';
+
+type TTimelineItemProps = HTMLAttributes<HTMLDivElement> & { itemTitle: React.ReactNode };
+
+type TTimelineProps = HTMLAttributes<HTMLOListElement> & {
+    children: ReactElement<TTimelineItemProps>[];
+    lineHeight?: ComponentProps<typeof Text>['lineHeight'];
+};
+
+const TimelineItem = ({ children }: PropsWithChildren<TTimelineItemProps>) => {
+    return <div>{children}</div>;
+};
+
+const Marker = ({ label }: { label: number }) => {
+    return (
+        <div className='border absolute w-24 h-24 pl-px border-solid-red-0 rounded-full bg-solid-red-0 leading-[2.35rem] mr-8 text-center -left-12'>
+            <Text className='relative text-white align-middle text-[1.6rem] leading-normal' size='md' weight='bold'>
+                {label}
+            </Text>
+        </div>
+    );
+};
+/**
+ * @deprecated TODO: Replace this component with the one from @deriv-com/ui is implemented.
+ */
+export const Timeline = ({ children, className, lineHeight }: TTimelineProps) => {
+    if (!Array.isArray(children)) return null;
+    return (
+        <ol className={twMerge('ml-[1.2rem]', className)}>
+            {children.map((child, idx) => {
+                return (
+                    <li
+                        className='relative mb-[0] ms-0 block w-full border-solid-red-0 border-l-1 border-solid last-of-type:border-l-0 pb-[1.6rem]'
+                        key={idx}
+                    >
+                        <Marker label={idx + 1} />
+                        <div className='ml-20 w-full'>
+                            <Text as='h2' className='max-w-[500px]' color='prominent' lineHeight={lineHeight} size='xs'>
+                                {child.props.itemTitle}
+                            </Text>
+                            <div className='my-16 mx-[0] text-system-light-prominent-text last-of-type:mb-[0]'>
```

Comment:
```suggestion
                            <div className='my-16 mx-0 text-system-light-prominent-text last-of-type:mb-0'>
```


As we have a class for `mx-0` so don't you think we should use that 😬